apps/src/applab/appStorage.js

```diff
@@ -299,3 +299,48 @@ var handlePopulateTable = function (onSuccess, onError) {
     onSuccess();
   }
 };
+
+/**
+ * Populates the channel kv store with initial data
```

Comment:
"channels kv" --> "key/value"

we tend to avoid new acronyms and abbreviations when possible.